sentry-android-core/src/main/java/io/sentry/android/core/internal/util/Debouncer.java

```diff
@@ -10,12 +10,18 @@ public class Debouncer {
 
   private final long waitTimeMs;
   private final @NotNull ICurrentDateProvider timeProvider;
+  private int executions = 0;
```

Comment:
`m`: I feel like this should be atomic (and also `lastExecutionTime` fwiw), because `checkForDebounce` can be called from multiple threads simultaneously in theory